src/plot_topoplotseries.jl

```diff
@@ -132,15 +135,11 @@ function plot_topoplotseries!(
         config.visual...,
         positions,
     )
-    if (config.colorbar.colorrange !== nothing)
-        config_kwargs!(config)
-    else
-        config_kwargs!(
-            config,
-            visual = (; colorrange = colorrange),
-            colorbar = (; colorrange = colorrange),
-        )
-    end
+    config_kwargs!(
```

Comment:
this simplification relies on eeg_topoplot_series! to apply the correct colorrange - are you sure one can still set their own colorrange?

Reply:
I checked, it is fine